rts/lra-examples/cdi-participant-with-coordinator/pom.xml

```diff
@@ -76,6 +95,17 @@
             <version>${project.version}</version>
             <type>war</type>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.microprofile.lra</groupId>
+            <artifactId>microprofile-lra-api</artifactId>
+            <version>${version.microprofile.lra}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.narayana.rts</groupId>
+            <artifactId>lra-service-base</artifactId>
```

Comment:
why does the client service need to explicitly depend on lra-service-base?

Reply:
It doesn't so I removed it